lib/rules/no-deprecated-router-transition-methods.js

```diff
@@ -76,26 +76,29 @@ module.exports = {
       if (emberUtils.isAnyEmberCoreModule(context, node)) {
         const classMembers = node.body.body;
 
-        for (const classMember of classMembers) {
-          if (emberUtils.isInjectedServiceProp(classMember, undefined, serviceInjectImportName)) {
-            const serviceExpression = decoratorUtils.findDecorator(
-              classMember,
-              serviceInjectImportName
-            ).expression;
-
-            if (serviceExpression.type === 'CallExpression') {
-              if (
-                (serviceExpression.arguments.length === 0 && classMember.key.name === 'router') ||
-                (serviceExpression.arguments.length > 0 &&
-                  serviceExpression.arguments[0].value === 'router')
-              ) {
+        if (serviceInjectImportName) {
```

Comment:
Without this if statement, if inject service wasn't imported: then any property would have been checked/considered as being a service.